xbmc/Application.h

```diff
@@ -229,14 +222,6 @@ friend class CAppInboundProtocol;
 
 protected:
   bool OnSettingsSaving() const override;
```

Comment:
Could you explain why this one was not factored out to `CApplicationSettingsHandling`?

Reply:
`CApplication` is no longer directly derived from `ISettingsHandler`, but inherits this interface from `CApplicationSettingsHandling`. This is why this overrode still works. Is this by intention?

Reply:
I see.